sound/soc/intel/boards/Kconfig

```diff
@@ -498,4 +498,22 @@ config SND_SOC_INTEL_TGL_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE
 
+if (SND_SOC_SOF_JASPERLAKE || SND_SOC_SOF_ICELAKE) && SND_SOC_SOF_HDA_LINK
+
+config SND_SOC_INTEL_JSL_DA7219_MAX98373_MACH
+	tristate "JSL with DA7219 and MAX98373 in I2S Mode"
+	depends on I2C && ACPI
+	depends on MFD_INTEL_LPSS || COMPILE_TEST
+	select SND_SOC_DA7219
+	select SND_SOC_MAX98373
+	select SND_SOC_DMIC
+	select SND_SOC_HDAC_HDMI
```

Comment:
@yongzhi1 hdac-hdmi support for ICL and JSL is dropped from upstream, so you need to you snd-hda-codec-hdmi. Please use

select SND_HDA_CODEC_HDMI if SND_SOC_SOF_HDA_CODEC

here.

Reply:
Ack, will switch to SND_HDA_CODEC_HDMI.